fuel/datasets/caltech101_silhouettes.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+import os
+
+from fuel import config
+from fuel.datasets import H5PYDataset
+
+
+class CalTech101Silhouettes(H5PYDataset):
+    u"""CalTech 101 Silhouettes dataset.
+
+    This dataset provides the `split1` train/validation/test split of the
+    CalTech101 Silhouette dataset prepared by Benjamin M. Marlin [MARLIN].
+
+    This class provides both the 16x16 and the 28x28 pixel sized version.
+
+    .. [MARLIN] https://people.cs.umass.edu/~marlin/data.shtml
+
+    Parameters
+    ----------
+    which_set : 'train' or 'valid' or 'test'
```

Comment:
I recently learned that when an argument must be chosen from a set of values the standard `numpydoc` syntax is

    which_set : {'train', 'valid', 'test'}

Could you change this line to reflect that?